app/src/main/java/de/danoeh/antennapod/fragment/preferences/ImportExportPreferencesFragment.java

```diff
@@ -44,16 +47,16 @@
     private static final String PREF_HTML_EXPORT = "prefHtmlExport";
     private static final String PREF_DATABASE_IMPORT = "prefDatabaseImport";
     private static final String PREF_DATABASE_EXPORT = "prefDatabaseExport";
-    private static final String DEFAULT_OPML_OUTPUT_NAME = "antennapod-feeds.opml";
+    private static final String DEFAULT_OPML_OUTPUT_NAME = "antennapod-feeds.%s.opml";
     private static final String CONTENT_TYPE_OPML = "text/x-opml";
-    private static final String DEFAULT_HTML_OUTPUT_NAME = "antennapod-feeds.html";
+    private static final String DEFAULT_HTML_OUTPUT_NAME = "antennapod-feeds.%s.html";
```

Comment:
I think it is strange to use a `.` here (as it is associated with file extensions). I suggest to use `antennapod-feeds-%s.html` instead (same for the other file names)